less/pageLevelProgressIndicator.less

```diff
@@ -0,0 +1,93 @@
+.menu-title-inner,
+.menu-item-title-inner,
+.page-title-inner,
+.article-title-inner,
+.block-title-inner,
+.component-title-inner {
+    display: inline-block;
+}
+
+.pagelevelprogress-indicator {
+    display: inline-block;
+    box-sizing: border-box;
+    height: 12px;
+    width: 24px;
+    border: 2px solid @item-color-hover;
+    border-radius: 50px;
+    background-color: @foreground-color-inverted;
+    vertical-align: middle;
+    overflow: hidden;
+
+    &.complete .pagelevelprogress-indicator-inner {
+        width: 100%;
+    }
+}
+
+.pagelevelprogress-indicator-inner {
+    position: relative;
+    overflow: hidden;
+    box-sizing: border-box;
+    height: 8px;
+    border: 1px solid @foreground-color-inverted;
+    border-radius: 25px;
+}
+
+.pagelevelprogress-indicator-bar {
+    position: absolute;
+    min-width: 4px;
+    height: 100%;
+    background-color: @item-color-hover;
+}
+
+.pagelevelprogress-navigation {
+    height: @navigation-height;
```

Comment:
Just a heads up: `@navigation-height` already exists as a legacy variable in `adapt-defaults.less` so does cause a height discrepancy when using an older version of Vanilla:

![height](https://user-images.githubusercontent.com/922987/46605209-a7254700-caf0-11e8-872b-7dc632a4bc66.png)

Reply:
Ta.

Reply:
I've done some digging and this is actually occurring because the **current** release of Vanilla doesn't [redefine](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/v3.0.0/less/src/navigation.less#L9) the [`@navigation-height`](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/v3.0.0/less/src/adapt-defaults.less#L339) variable, so the PLP Less added in this PR is referencing the wrong pixel value.

Are we planning on doing another release of Vanilla before PLP gets one? This [commit](https://github.com/adaptlearning/adapt-contrib-vanilla/commit/b998619345e577dfab44cb0b50899106beb9c454) is the clincher. Otherwise everyone faces the same bug.

Or should we be using padding rather than defined height? @guywillis